bokehjs/src/coffee/models/glyphs/webgl/line.coffee

```diff
@@ -670,7 +670,7 @@ class LineGLGlyph extends BaseGLGlyph
 
       # Do we need to re-calculate segment data and cumsum?
       if Math.abs(@_scale_aspect - (sy / sx)) > Math.abs(1e-3 * @_scale_aspect)
-        @_update_scale(sx, sy)
+        mainGlGlyph.update_scale(sx, sy)
         @_scale_aspect = sy / sx
```

Comment:
I think this should also use `mainGlGlyph._scale_aspect`. BTW IMO it's ok to use private props of the mainGlGlyph object since its the same class.